src/test/rustdoc-ui/intra-doc/global-path.stderr

```diff
@@ -0,0 +1,10 @@
+warning: unresolved link to `::Unresolved`
+  --> $DIR/global-path.rs:6:6
+   |
+LL | /// [::Unresolved]
+   |      ^^^^^^^^^^^^ no item named `` in scope
```

Comment:
Unrelated to the changes here, this error message seems wrong. Should we open an issue to change the [diagnostic](https://github.com/rust-lang/rust/blob/7f3e2c0d374ba7982ef09499c7de6bb53866abf4/src/librustdoc/passes/collect_intra_doc_links.rs#L916) to check for empty paths?

Reply:
Maybe.
I don't plan to work on this personally.

Reply:
I'm looking into it.